File: high_low.rb
array = (1..100).to_a.sample(50)

def high_low(ar)
    min = ar[0]
    max = ar[0]
    ar.each do |num|
        if num < min
            min = num
        elsif num > max
            max = num
        end
    end
    puts "High: #{max}\nLow: #{min}"
end
    
        

high_low(array)

File: high_low.js
let array = Array.from({length: 40}, () => Math.floor(Math.random() * 40));


function highLow(ar) {
  var min = ar[0]
  var max = ar[0]
  for (n in ar) {
    if (n < min) {
      min = n
    } else {
      max = n
    }
  }
  console.log(max)
  console.log(min)
}